### DISCUSSION TITLE
Video Download Quality set to 720p

### DISCUSSION THREAD

**Discussion by IMG20191023** [jmbannon/ytdl-sub#1120] [Q&A]
Hey, great app, works really well and thanks for your hard work !
I just have one question, i cannot for the life of me figure out how to reduce the downloaded video quality.
From what i understand there is a built in preset for Max_1080p ?
But i would like to download all my videos in 720p, when i set it to 720 the downloaded video file is still 1920x180 ! (Plex TV Show by Date | = max_720p:)

Any help? Thanks.

**Comment by tgrahl** [comment]
@IMG20191023 did you figure it out? I am also not having any luck

**Comment by IMG20191023** [comment]
I think my formatting was wrong.
I've moved to 1080 now, but you should be able to adjust the below line to whatever quality you want.

Plex TV Show by Date | Max 1080p:


The start of my yaml file I have the specific quality settings detailed, I can't remember if by default it comes with these or not.

#############################################################################
  # Max 1080p

  max_1080p:  # legacy name
    format: "(bv*[height<=1080]+bestaudio/best[height<=1080])"
    ytdl_options:
      merge_output_format: "mp4"

  "Max 1080p":
    preset:
      - max_1080p

  #############################################################################
  # Max 720p

  "Max 720p":
    format: "(bv*[height<=720]+bestaudio/best[height<=720])"
    ytdl_options:
      merge_output_format: "mp4"

  #############################################################################

Post your subscription ymal if it still doesn't work?

**Comment by jmbannon** [comment]
You can find the prebuilt 720p preset here:
https://ytdl-sub.readthedocs.io/en/latest/prebuilt_presets/media_quality.html#video

at the top is a hint containing a link on how to apply it properly.

**Reply by tgrahl** [reply]
Thank you that works great. Initially kept trying with "__preset__: overrides: format:",  but with the presets it works

Jellyfin TV Show by Date:
  = News | Only Recent | **Max 480p:**
